apps/nestjs-wallet/src/vc-api/exchanges/entities/transaction.entity.ts

```diff
@@ -75,6 +76,57 @@ export class TransactionEntity {
   @Column('simple-json')
   callback: CallbackConfiguration[];
 
+  private verifyVpRequestTypeDidAuth(presentation: VerifiablePresentation): void {
+    // https://w3c-ccg.github.io/vp-request-spec/#did-authentication-request
+    if (!presentation.holder) throw Error('Presentation holder is required for didAuth query');
+  }
+
+  private verifyVpRequestTypePresentationDefinition(
+    presentation: VerifiablePresentation,
+    credentialQuery: Array<{ presentationDefinition: IPresentationDefinition }>
+  ): void {
+    // https://identity.foundation/presentation-exchange/#presentation-definition
+    if (presentation.verifiableCredential.length !== this.vpRequest.query.length)
```

Comment:
I'm not sure this is necessarily the case. You could have a single `presentationDefinition` ask for multiple credentials in a presentation, right?

Reply:
Yeah, your are right. I'll update it